### DISCUSSION TITLE
Suppressing alerts help

### DISCUSSION THREAD

**Discussion by jcurtiss7492** [Security-Onion-Solutions/securityonion#3733] [Unsupported Versions] [answered]
I am just beginning my journey with Security Onion 2, so sorry for such a newbie question :)

I have a standalone install, all on one machine for a very small production network.  So far everything seems to work great, but I'm having an issue with suppressing alerts.

I am trying to follow the procedure outlined in:  https://docs.securityonion.net/en/2.3/managing-alerts.html#suppressions, but when I try to get access to /opt/so/saltstack/local/pillar/global.sls, I get the "-bash: cd: saltstack/: Permission denied".  In fact, my access seems nil once I get the the saltstack directory.  Noob me has tried using sudo, but no luck.  I am logged in with the admin account I created during install.

Can someone help steer me in the right direction?  Thanks!

**Comment by jdecker79** [comment, marked as answer]
Fellow noob here.  I think I ran into that and was able to get past it by running sudo su to get super user access.

**Reply by jcurtiss7492** [reply]
Thanks for the lesson, jdecker79.  su was golden ticket!
